test/test.manifests.js

```diff
@@ -317,3 +329,141 @@ describe('manifests', function(done) {
     });
   });
 });
+
+describe('validator manifests endpoint', function() {
+
+  before(function(done) {
+    manifests = new Manifests();
+    hbase.deleteAllRows({
+      table: 'manifests_by_master_key'
+    }).then(() => {
+      return hbase.deleteAllRows({
+        table: 'manifests_by_validator'
+      })
+    }).then(() => {
+      Promise.map(mockManifests, function(man) {
+        return manifests.handleManifest(man)
+      })
+    }).then(() => { done(); })
+  });
+
+  it('should get validator manifest', function(done) {
+
+    var pubkey = 'nHBV75zgMXCRHiuTMq6MdbcA6tBoSMWucTvHrnkQFW9gAXWoW15N'
+    var url = 'http://localhost:' + port +
+        '/v2/network/validators/' + pubkey + '/manifests'
+
+    request({
+      url: url,
+      json: true
+    },
+    function(err, res, body) {
+      assert.ifError(err)
+      assert.strictEqual(res.statusCode, 200)
+      assert.strictEqual(body.manifests.length, 1)
+      _.isMatch(body.manifests[0], mockResponses[pubkey][0])
+      done()
+    });
+  });
+
+  it('should get multiple validator manifests', function(done) {
+    var pubkey = 'nHDEmQKb2nbcewdQ1fqCTGcPTcePhJ2Rh6MRftsCaf6UNRQLv7pB'
+    var url = 'http://localhost:' + port +
+        '/v2/network/validators/' + pubkey + '/manifests'
+
+    request({
+      url: url,
+      json: true
+    },
+    function(err, res, body) {
+      assert.ifError(err)
+      assert.strictEqual(res.statusCode, 200)
+      assert.strictEqual(body.manifests.length, 2)
+      _.isMatch(body.manifests[0], mockResponses[pubkey][0])
+      _.isMatch(body.manifests[1], mockResponses[pubkey][1])
+      done()
+    });
+  });
+
+  it('should use limit and marker', function(done) {
+    var pubkey = 'nHDEmQKb2nbcewdQ1fqCTGcPTcePhJ2Rh6MRftsCaf6UNRQLv7pB'
+    var url = 'http://localhost:' + port +
+        '/v2/network/validators/' + pubkey + '/manifests'
+
+    request({
+      url: url + '?limit=1',
+      json: true
+    },
+    function(err, res, body) {
+      assert.ifError(err)
+      assert.strictEqual(res.statusCode, 200)
+      assert.strictEqual(body.manifests.length, 1)
+      _.isMatch(body.manifests[0], mockResponses[pubkey][0])
+      request({
+        url: url + '?marker=' + body.marker,
+        json: true
+      },
+      function(err, res, body) {
+        assert.ifError(err)
+        assert.strictEqual(res.statusCode, 200)
+        assert.strictEqual(body.manifests.length, 1)
+        _.isMatch(body.manifests[0], mockResponses[pubkey][1])
+        done()
+      })
+    });
+  });
+
+  it('should get validator revocation manifests', function(done) {
```

Comment:
whats this - "revocation manifests"? looks the same except descending order

Reply:
A manifest with a sequence of max unsigned int revokes the master key and doesn't need an ephemeral key/signature.
https://github.com/ripple/rippled-historical-database/blob/d35d45b0f6166c359c9bd2d3eb7432b0bfccc9d4/lib/validations/manifests.js#L257
I just shoehorned the descending into this test.